x/auth/ante/fee.go

```diff
@@ -83,26 +84,27 @@ func (dfd DeductFeeDecorator) checkDeductFee(ctx sdk.Context, sdkTx sdk.Tx, fee
 	}
 
 	feePayer := feeTx.FeePayer()
-	// feeGranter := feeTx.FeeGranter()
 	deductFeesFrom := feePayer
 
-	// TODO HV2: removed `FeeGranter` logic as we are not using it in heimdall. Is this ok? Do we want to use it instead of `FeeCollector`?
+	// TODO HV2: feeGranter set to nil as not used in heimdall
+	feeGranter := []byte(nil) // feeGranter := feeTx.FeeGranter()
```

Comment:
This seems potentially nefarious, given the sdk is supposed to be general purpose, I would not explicitly set this as nil here. I could be wrong but I think it's fine to just not pass a feegrant keeper to the handler options

Reply:
I checked the code, and it seems this should not be an issue, as used mostly in this file.
It is also used in `makeSignBatchCmd` which is not used in `heimdall` anyway.
Also, as a comment suggests `// FeeGranter always returns nil for StdTx` and if I am not mistaken, that's the tx type we are using.
Do you suggest not passing the `FeegrantKeeper` anyway to the `HandlerOptions`?